src/components/Toggle/index.tsx

```diff
@@ -0,0 +1,57 @@
+import React, { useState } from 'react'
+import {
+	Checkbox,
+	LeftLabel,
+	RightLabel,
+	RoundSlider,
+	Slider,
+	Switch,
+	Wrapper,
+} from './styled'
+
+interface Props {
+	onChange: (checked: boolean) => void
+	isChecked?: boolean
+	isRound?: boolean
+	label?: [string, string]
+	className?: string
+}
+
+const Toggle: React.FC<Props> = ({
+	onChange,
+	isChecked = false,
+	isRound = false,
+	label,
+	className,
+}) => {
+	const [checked, setChecked] = useState<boolean>(isChecked)
+
+	const change = (checked: boolean): void => {
+		setChecked(checked)
+		onChange(checked)
+	}
+
+	const [leftLabel, rightLabel] = label ?? ['', '']
+
+	return (
+		<Wrapper className={className}>
```

Comment:
"A `wrapper` is a function that is intended to call one or more other functions, sometimes purely for convenience, and sometimes adapting them to do a slightly different task in the process." - [MDN Web Docs](https://developer.mozilla.org/en-US/docs/Glossary/Wrapper)

Which case is ours? Also it seems to me a bit like Clojure macros, am I right?

Reply:
No, `Wrapper` is just custom styled-component that I made to wrap and style (in this case position) components contanied in here. It could be renamed to anything, this is just a convetion... :sweat_smile: